### PR TITLE
feat: Add iam_email and env_vars output to simple-sa submodule

### PR DESCRIPTION
**Pull request by q2w**
This PR makes below changes,

1. Adds new output `iam_email`
2. Adds new output  `env_vars`

### CODE REVIEW

File: modules/simple-sa/outputs.tf [hunk 1]
```diff
@@ -19,10 +19,21 @@ output "email" {
   value       = google_service_account.sa.email
 }
 
+output "iam_email" {
+  description = "IAM format service account email"
+  value       = "serviceAccount:${google_service_account.sa.email}"
```
Comment (bharathkkb) [inline modules/simple-sa/outputs.tf:24]: nit: you can directly use https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_service_account#member

File: modules/simple-sa/outputs.tf [hunk 2]
```diff
@@ -19,10 +19,21 @@ output "email" {
...
+
 output "id" {
   description = "Service account id and email"
   value = {
     id    = google_service_account.sa.account_id,
     email = google_service_account.sa.email
   }
 }
+
+output "env_vars" {
```
Comment (bharathkkb) [inline modules/simple-sa/outputs.tf:35]: Where do we plan to use these as env vars?

Comment (q2w) [reply modules/simple-sa/outputs.tf:35]: JSS solution three-tier-web-app and Gen-AI-RAG-with-cloud-sql both requires the SERVICE_ACCOUNT_EMAIL environment variable in backend service (cloud-run).
1. https://github.com/q2w/terraform-genai-rag/blob/using-adc-format/infra/main.tf#L80
2. https://github.com/q2w/solution-builder-three-tier-web-app/blob/apphub-prototype/infra/main.tf#L83

Adding this env_vars output to use it directly as env_vars in backend (cloud-run)

File: modules/simple-sa/outputs.tf (hunk 1 above)
Comment (q2w) [reply modules/simple-sa/outputs.tf]: Done. Thanks!

File: modules/simple-sa/outputs.tf (hunk 2 above)
Comment (bharathkkb) [reply modules/simple-sa/outputs.tf:35]: gotcha, sg

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**